docs/api-docs/storefront/widgets/widgets-localizing-settings.md

```diff
@@ -0,0 +1,214 @@
+# Localizing Widget Template Settings
+
+<div class="otp" id="no-index">
+
+### On this page
+
+- [Mapping settings to translations](#mapping-settings-to-translations)
+- [Formatting language codes](#formatting-language-codes)
+- [Resources](#resources)
+  
+</div>
+
+You can localize widget template settings to the provided language translations using the internationalization (**i18n**) system.
+
+This article explains how to localize your widget template settings and provides a practical example of how to map settings to translations. 
+
+## Mapping settings to translations
+
+The [widget template](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget-template/getwidgettemplates) schema uses the `i18n.{SettingName}` internationalization format within labels to map settings to translation files. i18n's interpolation functionality makes it possible to integrate dynamic values into translations.
+
+You can localize your widget template settings by defining translations in the `schema_translations` property of the widget template schema. 
+
+### Localization example
+
+The following example demonstrates how to add translations for the widget template settings using the `schema_translations` property. 
+
+1. Identify all of the settings that need a translation. Each `label` value must start with the `i18n.` prefix. For example, `"label": "i18n.LineColor"`.
+2. Prepare your schema translations JSON object. It must contain a default language code along with the other languages you want to support formatted by two lowercase letters, a dash, and at least two alphanumeric characters. For more information, see the [Formatting language codes](#formatting-language-codes) section below.
+
+**Schema translations example**
+
+```json
+{
+ "i18n.LineColor": {
+   "default": "Line color",
+   "fr-FR": "Couleur de la ligne",
+   "en": "Line color",
+   "zh-CH": "线条颜色"
```

Comment:
Hi Katya, would it be possible to change the chinese locale to `zh-CN`? Sorry for the miscommunication, but there was a very recent change to the way we are handling languageCodes and this should load the proper chinese setting.